src/test/kotlin/yij/ie/ideacommentqueries/providers/CommentCollectorKtTest.kt

```diff
@@ -1,13 +1,9 @@
-package yij.ie.ideacommentqueries.test.providers
+package yij.ie.ideacommentqueries.providers
 
-import org.junit.Test
-import yij.ie.ideacommentqueries.providers.defineAbsoluteMatcher
-import yij.ie.ideacommentqueries.providers.defineRelativeMatcherRegExp
-import yij.ie.ideacommentqueries.providers.resolveRelativeMatchResult
-import kotlin.test.assertEquals
+import com.intellij.testFramework.fixtures.BasePlatformTestCase
+
+internal class CommentCollectorKtTest  : BasePlatformTestCase() {
```

Comment:
```suggestion
internal class CommentCollectorKtTest : BasePlatformTestCase() {
```

Looks like a coding style error?